src/components/DocumentationLayout.vue

```diff
@@ -150,12 +152,21 @@ export default {
       sidenavHiddenOnLarge: storage.get(NAVIGATOR_HIDDEN_ON_LARGE_KEY, false),
       showQuickNavigationModal: false,
       BreakpointName,
+      indexState: IndexStore.state,
     };
   },
   computed: {
     enableQuickNavigation: ({ isTargetIDE }) => (
       !isTargetIDE && getSetting(['features', 'docs', 'quickNavigation', 'enable'], true)
     ),
+    quickNavData({ indexState, interfaceLanguage }) {
```

Comment:
Minor thing, but I think the "...Data" name is a little generic as to the shape of data it actually holds.

Maybe something like `indexNodes` or `navigatorNodes` would more clearly indicate that this prop will always resolve to the flat array of nodes?